examples/Microsoft.Spark.CSharp.Examples/MachineLearning/Sentiment/README.md

```diff
@@ -0,0 +1,188 @@
+
+One option is to use Model Builder's *Add Projects* feature, which will result in 3 projects:
+
+* Your original app (**myMLApp**)
+* A console app that allows you to build/train/test the model (**myMLAppML.ConsoleApp**)
+* A .NET Standard class library that contains model input/output and your trained model in a zip file (**myMLAppML.Model**)
+
+You would begin writing your .NET for Apache Spark code (and paste in the code generated from Model Builder, shown in step 3 above) in the original app **myMLApp.**
+
+![Model Builder Result](https://dotnet.microsoft.com/static/images/model-builder-generated-code.png?v=iC-r8k3zpKUwQVoNOH34D903IhXhIb4CsX003484s7c)
+
+#### Option 2: Create a new console app (shown in this repo)
+
+Rather than working with the projects/files produced by Model Builder's Add Projects, you can create a new, separate C# console app. You just need to copy over your model's zip file to a directory your new console app can access. In this repo, a trained model **MLModel.zip** has already been included for you in the *Resources* folder.
+
+As we create the logic for our Spark app, we'll paste in the code generated from Model Builder and include some other class definitions.
+
+## .NET for Spark
+
+Now that we've trained an ML.NET model for sentiment analysis, we can begin writing the .NET for Spark code that will read in our Yelp data, pass each review to our ML.NET model, and predict whether reviews are positive or negative.
+
+### 1. Create a Spark Session
+
+In any Spark application, we need to establish a new SparkSession, which is the entry point to programming Spark with the Dataset and 
+DataFrame API.
+
+```CSharp
+SparkSession spark = SparkSession
+       .Builder()
+       .AppName(".NET for Apache Spark Sentiment Analysis")
+       .GetOrCreate();
+```
+
+### 2. Read Input File into a DataFrame
+
+We trained our model with the Amazon data, so let's test how well the model performs by testing it with the Yelp dataset. 
+
+```CSharp
+DataFrame df = spark.Read().Csv(<Path to yelp data set>);
+```
+
+If we want to specify some other aspects of our data, such as whether it has a header and how we want to deal with its schema, we can set some other options when reading in our data:
+
+```CSharp
+DataFrame df = spark
+    .Read()
+    .Option("header", true)
+    .Option("inferSchema", true)
+    .Csv(<Path to yelp data set>);
+```
+
+### 3. Use a UDF to Access ML.NET
+
+We create a User Defined Function (UDF) that calls the *Sentiment* method on each Yelp review.
+
+```CSharp
+spark.Udf().Register<string, bool>("MLudf", (text) => Sentiment(text));
+```
+
+*Sentiment* is where we'll call our ML.NET code that was generated from the final step of Model Builder.
+
+```CSharp
+MLContext mlContext = new MLContext();
+ITransformer mlModel = mlContext.Model.Load(@"Resources\MLModel.zip", out var modelInputSchema);
+var predEngine = mlContext.Model.CreatePredictionEngine<Review, ReviewPrediction>(mlModel);
+```
```

Comment:
Slightly different than the code, but I assume it's to make it easier to understand ?

Reply:
I think it was initially to make it easier, but it just seems like now it'd be easier to include and explain the actual code. I've updated it to reflect the current code in `Program.cs`